thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detector/email/filter/TestBaseAlertFilter.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.thirdeye.detector.email.filter;
+
+import com.linkedin.thirdeye.datalayer.bao.AbstractManagerTestBase;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import java.util.Map;
+import org.junit.Test;
+import org.testng.Assert;
+
+
+
+public class TestBaseAlertFilter extends AbstractManagerTestBase {
+  private static String collection = "my dataset";
+  private static String metricName = "__counts";
+
+  // test set up double, Double, string for alpha_beta
+  // also test missing input parameter, will use default value defined in the specified class
+  @Test
+  public void testSetAlphaBetaParamter(){
+    AnomalyFunctionDTO anomalyFunctionSpec = getTestFunctionAlphaBetaAlertFilterSpec(metricName, collection);
+    Map<String, String> alertfilter = anomalyFunctionSpec.getAlertFilter();
+    AlphaBetaAlertFilter alphaBetaAlertFilter = new AlphaBetaAlertFilter();
+    alphaBetaAlertFilter.setParameters(alertfilter);
+    Assert.assertEquals(alphaBetaAlertFilter.getAlpha(), Double.valueOf(alertfilter.get("alpha")));
+    Assert.assertEquals(alphaBetaAlertFilter.getBeta(), Double.valueOf(alertfilter.get("beta")));
+    Assert.assertEquals(alphaBetaAlertFilter.getType(), alertfilter.get("type"));
+    Assert.assertEquals(alphaBetaAlertFilter.getThreshold(), Double.valueOf(alertfilter.get("threshold")));
+
+    // test scientific decimal
+    double threshold = 1E-10;
+    alertfilter.put("threshold", String.valueOf(threshold));
+    alphaBetaAlertFilter.setParameters(alertfilter);
+    Assert.assertEquals(alphaBetaAlertFilter.getThreshold(), Double.valueOf(alertfilter.get("threshold")));
+
+    // test missing field
```

Comment:
Since within alertfilter class, for example AlphaBetaAlertFilter, the private variables are initially being assigned as DEFAULT value, thus here it will be equal to default value when missing